src/main/java/gov/cabinetoffice/gap/applybackend/web/JwtController.java

```diff
@@ -76,4 +78,31 @@ public ResponseEntity<IsAdminJwtResponse> isAdmin(@RequestHeader("Authorization"
 
         return ResponseEntity.ok(response);
     }
+
+    @GetMapping("/userRoles")
+    public ResponseEntity<UserRolesJwtResponse> userRoles(@RequestHeader("Authorization") String jwtToken)
+            throws JwkException {
+
+        if (jwtToken.length() <= 0) {
+            throw new JwtTokenUndefinedException("No Jwt has been passed in the request");
+        }
+
+        final String normalisedJwt = jwtToken.split(" ")[1];
+        final DecodedJWT jwt = jwtService.decodedJwt(normalisedJwt);
+        final boolean isValid = jwtService.verifyToken(normalisedJwt);
+        final JwtPayloadV2 payload = jwtService.decodeTheTokenPayloadInAReadableFormatV2(jwt);
+
+        final boolean isSuperAdmin = payload.getRoles().contains("SUPER_ADMIN");
+        final boolean isAdmin = payload.getRoles().contains("ADMIN");
```

Comment:
We get a json string array like this:

```
roles: "[ADMIN, APPLICANT]"
```

we could do with converting this into a `List` of the individual entries, as the `SUPER_ADMIN` role at present will always produce a hit against the `ADMIN` role.